experiment/coverage/apicoverage.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/go-openapi/spec"
+	"github.com/golang/glog"
+)
+
+var (
```

Comment:
A useful future improvement might be a `--minimumCoverage=0.5` or something, and then at the end of the main function fail if the number of apis coverered is less than 50% 

Then we can dump this into a regression test

Reply:
Cool, that might be useful for operating this tool on CI system to notify warning of coverage.
I think we will have some points we want to discuss related to this feature.
So I will register this point as an issue after first merger.

Reply:
This is registered as https://github.com/kubernetes/test-infra/issues/6079